proposals/0173-Read-Generic-Network-Signal-data.md

```diff
@@ -63,53 +71,69 @@ A base JSON schema file for SDL core and proxy will be kept locally and an addit
 
 ```
-"vehicle_data_schema_version":"00.00.01",
+"schema_version":"00.00.01",
 ```
 
-#### Versioning and endpoint for OEM mapping table and to provide HMI a way to read OEM mapping table version
+#### Versioning and endpoint for OEM Network Mapping table and to provide HMI a way to read OEM Network Mapping table version
 
-OEM mapping table version would be included in PTU, which SDL will persist in local policy DB. HMI needs to be able to read this value on demand so that it can control if and when to download the OEM mapping table. HMI will utilize the _endpoint_ for OEM mapping table file to download this file using _SystemRequest_ type _OEM_SPECIFIC_. OEM mapping table file would have _endpoint_ key (_service_) as _oem_mapping_file_.
+OEM Network Mapping table version would be included in PTU, which SDL core will persist in local policy DB. HMI needs to be able to read this value on demand so that it can control if and when to download the OEM Network Mapping table. HMI will utilize the _endpoint_ for OEM Network Mapping table file to download this file using _SystemRequest_ `requestType` _OEM_SPECIFIC_ and  `requestSubType` _VEHICLE_DATA_MAPPING_. OEM Network Mapping table file would have _endpoint_ key (_service_) as _0x0A_.
 
-SDL still needs to provide an API for HMI to read this value. We can do so by adding a new API to read OEM mapping file version; we can make it generic so that it can also be used in the future to read some other configuration params from policy. e.g. following new API can be used to read OEM mapping table version with _identifier_ _oem_mapping_version_
+SDL core still needs to provide an API for HMI to read this value. We can do so by adding a new API to read OEM Network Mapping file version; we can make it generic so that it can also be used in the future to read some other configuration params from policy. e.g. following new API can be used to read OEM Network Mapping table version with _identifier_ `oem_mapping_version`
 
 ```
 <function name="GetPolicyConfigurationData" messagetype="request" scope="internal">
-	<description>Request from HMI to SDL to get policy configuration data (i.e. OEM mapping table file version etc.) from Policy Table.</description>
-	<param name="identifier" type="String" array="true" minsize="1" maxsize="100" maxlength="500" mandatory="true">
-		<description>Id of configuration data to be received according to Policy Table i.e. oem_mapping_version etc.</description>
+	<description>Request from HMI to SDL core to get policy configuration data (i.e. OEM Network Mapping table file version etc.) from Policy Table.</description>
+	<param name="policyType" type="String" maxlength="1000" minlength="1" mandatory="true">
+		<description>Name of the Struct from where configuration data to be received according to Policy Table types i.e. module_config etc.</description>
```

Comment:
>Name of the Struct from where configuration data to be received according to Policy Table types i.e. module_config etc.

Please update to "Name of the Struct where configuration data is located in Policy Table, i.e. module_config etc."

Reply:
Done